nautobot/extras/api/serializers.py

```diff
@@ -535,6 +535,8 @@ def get_parent(self, obj):
 
 
 class JobSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
+    task_queues = serializers.JSONField(read_only=True, required=False)
```

Comment:
Maybe add some comments to the serializers and graphql types documenting that these fields are for backward compatibility with 2.3 and earlier, just in case we wonder later.